http/data.go

```diff
@@ -49,6 +49,8 @@ func (d *data) Check(path string) bool {
 
 func handle(fn handleFunc, prefix string, store *storage.Storage, server *settings.Server) http.Handler {
 	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Cache-Control", "no-cache, no-store, must-revalidate")
```

Comment:
It doesn't make sense to set no-cache and no-store together. This is the correct headers to wipe existing and prevent to cache upcoming responses.
```suggestion
		w.Header().Set("Cache-Control", "no-store, max-age=0")
```